projects/1.16/assets/prehistoric-fauna/prehistoricfauna/lang/zh_cn.json

```
@@ -0,0 +1,968 @@
+	"prehistoricfauna.subtitle.henosIdle": "埃诺斯：呻吟",
+	"prehistoricfauna.subtitle.henosLaser": "时间激光：发射",
+	"prehistoricfauna.subtitle.henosHurt": "埃诺斯：受伤",
+	"prehistoricfauna.subtitle.henosDeath": "埃诺斯：死亡",
+	"prehistoricfauna.subtitle.protoceratopsIdle": "原角龙：？",
+	"prehistoricfauna.subtitle.protoceratopsHurt": "原角龙：受伤",
+	"prehistoricfauna.subtitle.protoceratopsDeath": "原角龙：死亡",
+	"prehistoricfauna.subtitle.protoceratopsWarning": "原角龙：？",
+	"prehistoricfauna.subtitle.velociraptorIdle": "伶盗龙：咕嘟",
+	"prehistoricfauna.subtitle.velociraptorHurt": "伶盗龙：受伤",
+	"prehistoricfauna.subtitle.velociraptorDeath": "伶盗龙：死亡",
+	"prehistoricfauna.subtitle.velociraptorWarning": "伶盗龙：？",
+	"prehistoricfauna.subtitle.citipatiIdle": "葬火龙：？",
+	"prehistoricfauna.subtitle.citipatiHurt": "葬火龙：受伤",
+	"prehistoricfauna.subtitle.citipatiDeath": "葬火龙：死亡",
+	"prehistoricfauna.subtitle.citipatiWarning": "葬火龙：？",
+	
+	"effect.prehistoricfauna.bleeding": "流血",
+	
+	"container.paleoscribe.no_paleojournal": "无法增加日志书页",
+	"container.paleoscribe.costs": "需要：",
+	"container.paleoscribe.paleopage.many": "%s日志书页",
+	
+	"advancements.prehistoricfauna.anti_lorax.title": "万事乐",
+	"advancements.prehistoricfauna.anti_lorax.description": "从每棵史前树木上收集木材",
+	"advancements.prehistoricfauna.cretaceous_time_travel.title": "总有一只霸王龙",
+	"advancements.prehistoricfauna.cretaceous_time_travel.description": "拜访白垩纪",
+	"advancements.prehistoricfauna.digger_man.title": "Digger Man",
+	"advancements.prehistoricfauna.digger_man.description": "挖到化石",
+	"advancements.prehistoricfauna.dinoe_exotic.title": "异国情调的恐龙",
+	"advancements.prehistoricfauna.dinoe_exotic.description": "培育每一种史前动物",
+	"advancements.prehistoricfauna.extinction_event.title": "你这灭绝元凶",
+	"advancements.prehistoricfauna.extinction_event.description": "杀死每一种史前动物",
+	"advancements.prehistoricfauna.fragile.title": "'Fragile', Must Be Italian!",
+	"advancements.prehistoricfauna.fragile.description": "获得每一个史前生物蛋",
+	"advancements.prehistoricfauna.frankenstein.title": "科学怪人",
+	"advancements.prehistoricfauna.frankenstein.description": "利用多种动物制作盔甲",
+	"advancements.prehistoricfauna.jurassic_time_travel.title": "泰坦纪元",
+	"advancements.prehistoricfauna.jurassic_time_travel.description": "拜访侏罗纪",
+	"advancements.prehistoricfauna.late_thag_simmons.title": "为萨格·西蒙斯报仇",
+	"advancements.prehistoricfauna.late_thag_simmons.description": "杀死一只剑龙",
+	"advancements.prehistoricfauna.looting_like_year.title": "与2020年相同的掠夺",
+	"advancements.prehistoricfauna.looting_like_year.description": "找到一间时间旅行者的破屋",
+	"advancements.prehistoricfauna.plagiarism.title": "剽窃",
+	"advancements.prehistoricfauna.plagiarism.description": "窃取时间旅行者的作品",
+	"advancements.prehistoricfauna.return_to_monke.title": "重新变成猩猩",
+	"advancements.prehistoricfauna.return_to_monke.description": "与埃诺斯战斗",
+	"advancements.prehistoricfauna.root.title": "史前生物",
+	"advancements.prehistoricfauna.root.description": "准备好时间旅行了吗，孩子们？",
+	"advancements.prehistoricfauna.stolen_guide.title": "搭便车回古代指南",
```

```suggestion
	"advancements.prehistoricfauna.stolen_guide.title": "古代漫游指南",
```
我觉得是捏他了《银河系漫游指南The Hitchhiker's Guide to the Galaxy》。